extension-manifest-v2/src/background.js

```diff
@@ -1569,21 +1569,31 @@ async function initializeAccount() {
 	}, 5000);
 
 	try {
-		await account.migrate();
-		lastStep = 'migrate';
+		try {
+			await account.getUser();
```

Comment:
This change is not obvious to me: there was a "migrate" before, which did multiple things including modifications to local storage. As I understood, the migrate functionality is no longer needed.

"getUser" is called for its side-effect, I guess. Maybe change the comment in the exception to something like "expected if the user is not logged in".